exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -211,4 +211,23 @@ public void testReversingNonEmptyList() {
                 ListOps.reverse(Arrays.asList('1', '3', '5', '7')));
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testListOfListIsNotFlattened() {
+        List<List<Character>> listOfLists = Arrays.asList(
+                Arrays.asList('1', '2'),
+                Collections.singletonList('3'),
+                Collections.emptyList(),
+                Arrays.asList('4', '5', '6'));
+
+        assertEquals(
+                Arrays.asList(
+                        Arrays.asList('4', '5', '6'),
+                        Collections.emptyList(),
+                        Collections.singletonList('3'),
+                        Arrays.asList('1', '2')
+                ),
```

Comment:
Very minor thing, but I think it's good to be consistent about either having the closing bracket on the same line or on a new line. I personally prefer on a new line, as I think it's more readable, but many people prefer it on the same line. Either is fine, as long as it's consistent.

So I'd prefer if we either move this closing bracket to the previous line, or change this file to have all closing brackets on a new line (though only when expressions are split over several lines of course). I have a slight preference for the second option, but either is fine. Does that make sense? 🙂

Reply:
I changed it to new line.